core/network/helpers/stream_read_buffer.hpp

```diff
@@ -151,6 +151,7 @@ namespace kagome::network {
     void read(libp2p::BytesOut out,
               size_t bytes,
               ReadCallbackFunc cb) override {
+      totalBytesRead += bytes;
```

Comment:
Move counter to separate libp2p header (`extern std::atomic_uint64_t foo;` or `inline auto &foo() { static std::atomic_uint64_t x = 0; return x }`)

Must count libp2p `TcpConnection` reads/writes